### ISSUE TITLE
Gnome-shell breaks when try lock screen. (crlt+esc or from suspend)

### ISSUE THREAD

**Issue by talesgomes27** [pop-os/gnome-shell#34]
Gnome-shell get a segment fault when lock screen should appear. 

Displays the following error on display.service:

```
mai 27 08:36:48 pop-os gdm-password][4942]: pam_unix(gdm-password:auth): Couldn't open /etc/securetty: Arquivo ou diretório inexistente
mai 27 08:36:57 pop-os gdm-password][4942]: pam_unix(gdm-password:auth): Couldn't open /etc/securetty: Arquivo ou diretório inexistente
mai 27 08:36:57 pop-os gdm-password][4942]: gkr-pam: unable to locate daemon control file
mai 27 08:36:57 pop-os gdm-password][4942]: gkr-pam: stashed password to try later in open session
mai 27 08:36:57 pop-os gdm-password][4942]: pam_unix(gdm-password:session): session opened for user tales by (uid=0)
mai 27 08:36:58 pop-os gdm-password][4942]: gkr-pam: gnome-keyring-daemon started properly and unlocked keyring
mai 27 09:59:17 pop-os gdm3[1164]: GLib: Source ID 69 was not found when attempting to remove it
mai 27 10:05:23 pop-os gdm3[1164]: GLib: Source ID 72 was not found when attempting to remove it
mai 27 10:11:07 pop-os gdm3[1164]: GLib: Source ID 76 was not found when attempting to remove it
mai 27 10:12:44 pop-os gdm3[1164]: GLib: Source ID 79 was not found when attempting to remove it
```

and on dmesg:

```
[ 4210.026410] sd 0:0:0:0: [sda] Starting disk
[ 4285.027632] sd 0:0:0:0: [sda] Synchronizing SCSI cache
[ 4285.029244] sd 0:0:0:0: [sda] Stopping disk
[ 4626.025247] ata1: SATA link up 6.0 Gbps (SStatus 133 SControl 300)
[ 4626.039723] ata1.00: ACPI cmd ef/10:06:00:00:00:00 (SET FEATURES) succeeded
[ 4626.039726] ata1.00: ACPI cmd f5/00:00:00:00:00:00 (SECURITY FREEZE LOCK) filtered out
[ 4626.039727] ata1.00: ACPI cmd b1/c1:00:00:00:00:00 (DEVICE CONFIGURATION OVERLAY) filtered out
[ 4627.781323] ata1.00: ACPI cmd ef/10:06:00:00:00:00 (SET FEATURES) succeeded
[ 4627.781325] ata1.00: ACPI cmd f5/00:00:00:00:00:00 (SECURITY FREEZE LOCK) filtered out
[ 4627.781326] ata1.00: ACPI cmd b1/c1:00:00:00:00:00 (DEVICE CONFIGURATION OVERLAY) filtered out
[ 4627.818486] ata1.00: configured for UDMA/133
[ 4627.827722] sd 0:0:0:0: [sda] Starting disk
[ 4650.616467] gnome-shell[60975]: segfault at 0 ip 00007f12219ae24f sp 00007fff8c2eeaa0 error 4 in libst-1.0.so[7f122198e000+4c000]
[ 4650.616472] Code: 00 00 00 48 83 c3 01 41 39 9f 30 01 00 00 0f 8e 27 01 00 00 49 8b 87 28 01 00 00 48 8d 35 ee 07 03 00 48 8b 2c d8 48 8b 45 00 <48> 8b 00 4c 8b 20 4c 89 e7 e8 63 31 fe ff 85 c0 74 c7 41 80 7c 24
[ 4661.393476] gnome-shell[75374]: segfault at 0 ip 00007f2f3aa7a24f sp 00007ffecf189c90 error 4 in libst-1.0.so[7f2f3aa5a000+4c000]
[ 4661.393483] Code: 00 00 00 48 83 c3 01 41 39 9f 30 01 00 00 0f 8e 27 01 00 00 49 8b 87 28 01 00 00 48 8d 35 ee 07 03 00 48 8b 2c d8 48 8b 45 00 <48> 8b 00 4c 8b 20 4c 89 e7 e8 63 31 fe ff 85 c0 74 c7 41 80 7c 24
```

![Captura de tela de 2020-05-27 10-29-06](https://user-images.githubusercontent.com/7399073/83026895-cd6d6280-a005-11ea-8717-8222bf7f83bd.png)
![Captura de tela de 2020-05-27 10-28-56](https://user-images.githubusercontent.com/7399073/83026915-cf372600-a005-11ea-969e-6919a6a0e6eb.png)

**Comment by talesgomes27** [comment]
I find out that it is a bug of the extension Dash to panel, not gnome-shell problem.

https://github.com/home-sweet-gnome/dash-to-panel/issues/1026#issue-626143894

**Comment by lehjr** [comment]
> I find out that it is a bug of the extension Dash to panel, not gnome-shell problem.
> 
> [home-sweet-gnome/dash-to-panel#1026 (comment)](https://github.com/home-sweet-gnome/dash-to-panel/issues/1026#issue-626143894)

actually, it is a bug with the current version of gnome-shell and is fixed in 3.36.3, but we probably won't see that version for awhile.
